src/Components/Results/Comparison.js

```diff
@@ -0,0 +1,63 @@
+const Comparison = ({ userWeather, uncleWeather, weatherSelection }) => {
+  const {
+    temperatureSelected,
+    cloudSelected,
+    windSelected,
+    humiditySelected,
+    twilightSelected
+  } = weatherSelection
+
+  const comparison = calcDiff(userWeather, uncleWeather);
+
+  function calcDiff(weatherObj1, weatherObj2) {
+    const diffObj = {};
+    for (let prop in weatherObj2) {
+      if (typeof (weatherObj2[prop]) === "number") {
+        diffObj[prop] = weatherObj2[prop] - weatherObj1[prop];
+      }
+    }
+    return diffObj;
+  }
+
+  return (
+    <div className="comparison">
+      <h3>Comparison</h3>
+      <ul>
+        {
+          !temperatureSelected ? null : (
+            <>
+              <li>Your Uncle is {Math.round(Math.abs(comparison.temp) * 100) / 100}°C {
```

Comment:
Maybe I'm missing something here, or with the way math.round / math.abs works, but I'm fairly sure multiplying by 100 and then dividing by 100 here is redundant